src/mon/Elector.h

```diff
@@ -357,6 +357,10 @@ class Elector : public ElectionOwner, RankProvider {
    * @post  @p participating is true
    */
   void start_participating();
+  /**
+  * Check if our peer_tracker is clean
```

Comment:
s/is clean/is self-consistent, not suffering from [tracker URL]/

Reply:
Addressed in new push